build.gradle

```diff
@@ -145,7 +145,7 @@ dependencies {
     androidTestImplementation 'com.android.support:support-annotations:25.4.0'
     compileOnly 'com.squareup.dagger:dagger-compiler:1.2.5'
     implementation 'com.github.wseemann:FFmpegMediaMetadataRetriever:1.0.14'
-
+    implementation 'com.google.android.exoplayer:exoplayer:2.7.3'
```

Comment:
You have included the full exoplayer library. This could be the reason for the build failing on travis, as the max strings allowed in dex is exceeding than the defined limit. As per your code looks like you require only core functionality.
```
implementation 'com.google.android.exoplayer:exoplayer-core:2.7.3'
```

Reply:
I have tried it. But still, it is giving the same error. Using 
`dexOptions {
        jumboMode true
    }`
seems to solve the problem.

Reply:
That was the last resort. Anyway, still reduce the dependency to `core` only (might save us some apk size).

Reply:
ok, I will change the dependency to include exoplayer-core. So, using `dexOptions { jumboMode true }` would be fine?

Reply:
yes